domino/domino.py

```diff
@@ -255,8 +261,9 @@ def job_start(self, command: str, commit_id: str = None, hardware_tier_name: str
                                                     The environment id to launch job with. If not provided
                                                     it will use the default environment for the project
         :param on_demand_spark_cluster_properties:  dict (Optional)
-                                                    On demand spark cluster properties. Following properties
-                                                    can be provided in spark cluster
+                                                    On demand spark cluster properties. Deprecated by
+                                                    compute_cluster_properties. Please use compute_cluster_properties
```

Comment:
The "by" clause seems redundant and grammatically questionable.
```suggestion
                                                    On demand spark cluster properties. Deprecated.
                                                    Please use compute_cluster_properties
```

Reply:
ok, updated